packages/flutter/test/widget/modal_barrier_test.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:test/test.dart';
+
+void main() {
+  bool tapped;
+  Widget tapTarget;
+
+  setUp(() {
+    tapped = false;
+    tapTarget = new GestureDetector(
+      child: new SizedBox(
+        width: 10.0,
+        height: 10.0,
+        child: new Text('Behind barrier')
+      ),
+      onTap: () {
```

Comment:
Put the onTap before child. The convention is to put the children last so that if the descendants are long, the other arguments are still close to the class name. Otherwise it isn't clear what they relate to.